metricsanalyzer/SAXHandler.java

```diff
@@ -0,0 +1,133 @@
+/*
+ * SAXHandler.java
+ *
+ */
+
+package metricsanalyzer;
+
+import org.xml.sax.helpers.DefaultHandler;
+import org.xml.sax.Attributes;
+import java.util.Vector;
+
+/**
+ *
+ * @author Byron
+ */
+public class SAXHandler extends DefaultHandler{
+    Vector metricsList = new Vector();
+    Vector systemMetrics = new Vector();
+    Vector classMetrics = new Vector();
+    Vector methodMetrics = new Vector();
+    Vector packageMetrics = new Vector();

```

Comment:
Ambiguous Information (A) - Vectors declared without types will generate warnings at compile time (lines 17 - 21)

Rank 2 - Fix as part of planned debugging sessions